src/component/contents/DraftEditorBlock.react.js

```diff
@@ -200,12 +200,11 @@ class DraftEditorBlock extends React.Component<Props> {
           UnicodeBidi.getDirection(decoratedText),
           this.props.direction,
         );
-
         const commonProps: DraftDecoratorComponentProps = {
           contentState: this.props.contentState,
           decoratedText,
           dir: dir,
-          key: decoratorOffsetKey,
+          key: entityKey || decoratorOffsetKey,
```

Comment:
I think this assumes that there's a 1:1 mapping between entities and decorations. We had the same problem and  ended up switching to `decoratorKey` instead in our fork (which has diverged quite a bit jfyi).

Reply:
Thanks for the review,it seems that using `decoratorKey` is more suitable and reasonable.
Should i add a commit that using `decoratorKey` or close this PR?